core/tests/webdriverio_utils/forms.js

```diff
@@ -250,6 +249,7 @@ var RichTextEditor = async function(elem) {
       await _clickToolbarButton(
         'cke_button__oppia' + componentName.toLowerCase());
 
+      var modalDialogElements = await $$('.modal-dialog');
```

Comment:
Why this was moved down and also why being an all selector it's not being referenced by a function?

Reply:
Done.